Examples/DicomImagePrintTags/Documentation.rst

```diff
@@ -32,3 +32,17 @@ Code
     .. literalinclude:: ../../Examples/DicomImagePrintTags/DicomImagePrintTags.R
        :language: r
        :lines: 18-
+
+Sample Output
+-------------
+
+Running the above Python script with the image  "docs/images/Image0075.dcm" produces the following output:
+
+.. exec_code::
+
+    # --- hide: start ---
+    import subprocess
+    result = subprocess.call(['python',
```

Comment:
Please see this change for a better way to get the current python executable.
64bd1f84